linetools/spectra/xspectrum1d.py

```diff
@@ -742,22 +783,24 @@ def write_to_fits(self, outfil, clobber=True, add_wave=False):
           Name of the FITS file
         clobber : bool (True)
           Clobber existing file?
-        add_wave : bool (False)
-          Force writing of wavelengths as array, instead of using FITS
-          header keywords to specify a wcs.
-        '''
+        """
         # TODO
         #  1. Add unit support for wavelength arrays
 
-        from specutils.wcs.specwcs import Spectrum1DPolynomialWCS, Spectrum1DLookupWCS
-        from specutils.io import write_fits as sui_wf
-        prihdu = sui_wf._make_hdu(self.data)  # Not for binary table format
-        prihdu.name = 'FLUX'
+        # Dummy hdu
+        prihdu = fits.PrimaryHDU()
+        prihdu.header['NSPEC'] = self.nspec
+        prihdu.header['NPIX'] = self.npix
 
-        hdu = fits.HDUList([prihdu])
+        #
+        tblhdu = fits.BinTableHDU(self._data, name='DATA')
+        # List
+
+        hdulist = fits.HDUList([prihdu, tblhdu])
 
+        """
         # Type
-        if type(self.wcs) is Spectrum1DPolynomialWCS:  # CRVAL1, etc. WCS
+        if type(self.wcs) is False:#Spectrum1DPolynomialWCS:  # CRVAL1, etc. WCS
```

Comment:
We can just pull out this WCS stuff altogether right? i.e. we will only support writing an array of wavelengths.